completions/asdf.fish

```diff
@@ -132,4 +132,5 @@ complete -f -c asdf -n '__fish_asdf_using_command shell; and test (count (comman
 
 # misc
 complete -f -c asdf -n '__fish_asdf_needs_command' -l "help" -d "Displays help"
+complete -f -c asdf -m '__fish_asdf_needs_command' -l "info" -d "Print OS, Shell and ASDF debug information"
```

Comment:
Just realized I missed this one, `-m` should have been `-n`. Not worth a PR so I pushed the fix directly to master.

Reply:
good catch! Thanks @danhper :100: